Examples/AmplitudeObjCExample/AmplitudeObjCExample/AppDelegate.m

```diff
@@ -0,0 +1,85 @@
+#import "AppDelegate.h"
+@import AmplitudeSwift;
+
+@interface AppDelegate ()
+
+@end
+
+@implementation AppDelegate
+
+
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+    NSString* apiKey = @"API-KEY";
+    AMPConfiguration* configuration = [AMPConfiguration initWithApiKey:apiKey];
+    configuration.logLevel = AMPLogLevelLOG;
+    configuration.serverZone = AMPServerZoneUS;
+    configuration.defaultTracking = AMPDefaultTrackingOptions.ALL;
+    configuration.loggerProvider = ^(NSInteger logLevel, NSString* _Nonnull message) {
+        NSLog(@"%@", message);
+    };
+    Amplitude* amplitude = [Amplitude initWithConfiguration:configuration];
+    
+    [amplitude add:[AMPPlugin initWithType:AMPPluginTypeBefore execute:^AMPBaseEvent* _Nullable(AMPBaseEvent* _Nonnull event) {
+        [event.eventProperties set:@"plugin-prop" value:@234];
+        event.locationLat = 34;
+        event.locationLng = 78;
+        return event;
+    }]];
+    
+    [amplitude setUserId:@"User-ObjC"];
+    
+    AMPIdentify* identify = [AMPIdentify new];
+    [identify set:@"user-prop-1" value:@"value-1"];
+    [identify set:@"user-prop-2" value:@123];
+    [amplitude identify:identify];
+    
+    [amplitude setGroup:@"orgName" groupName:@"Test Org"];
+    
+    AMPIdentify* groupIdentify = [AMPIdentify new];
+    [groupIdentify set:@"group-prop-1" value:@"value-A"];
+    [groupIdentify set:@"group-prop-2" value:@true];
+    [amplitude groupIdentify:@"orgName" groupName:@"Test Org" identify:groupIdentify];
+
+    [amplitude track:@"Event-A" eventProperties:@{
+        @"prop-string": @"value-A",
+        @"prop-int": @111,
+        @"prop-string-array": @[@"item-1", @"item-2"]
+    }];
+    
+    AMPBaseEvent* event = [AMPBaseEvent initWithEventType:@"Event-B"];
+    event.appVersion = @"1.2.3";
+    
+    [amplitude track:event callback:^(AMPBaseEvent* _Nonnull event, NSInteger code, NSString* _Nonnull message) {
+        NSLog(@"%@ - %@", event.eventType, message);
+    }];
+    
+    AMPBaseEvent* deepLinkOpenedEvent = [AMPDeepLinkOpenedEvent initWithUrl:@"http://example.com" referrer:@"https://referrer.com"];
+    [amplitude track:deepLinkOpenedEvent];
+    
+    AMPBaseEvent* screenViewedEvent = [AMPScreenViewedEvent initWithScreenName:@"Settings"];
```

Comment:
Not sure why but I am seeing this error:

<img width="837" alt="image" src="https://github.com/amplitude/Amplitude-Swift/assets/5790872/032297ab-6bde-498e-a38f-a0c78c4a2e4a">

Reply:
@justin-fiedler Hope it helps:
![image](https://github.com/amplitude/Amplitude-Swift/assets/9509929/3c930b1b-a7f9-44ba-97c7-2d074ac50420)